src/lib/cm/prompts/createCandyMachineWizardPrompt.ts

```
@@ -0,0 +1,315 @@
+        if (typeof val === 'string' && val.trim().toLowerCase() === 'q') {
+            console.log('Aborting wizard by user request.')
+            process.exit(0)
+        }
+        if (Array.isArray(val) && val.includes('Quit')) {
+            console.log('Aborting wizard by user request.')
+            process.exit(0)
+        }
+    }
+
+    // Prompt for directory-friendly name
+    let dirName = await input({
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dirName' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)